src/main/java/backend/resource/TurboIssue.java

```diff
@@ -337,6 +338,28 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return Optional.empty();
     }
 
+    /**
+     * Matching is done by matching all words separated by space in query
+     * @param issues
+     * @param query
+     * @return list of issues that contains the query
+     */
+    public static List<TurboIssue> getMatchedIssues(List<TurboIssue> issues, String query){
+        List<String> queries = Arrays.asList(query.split("\\s"));
+        return issues.stream()
+            .filter(i -> Utility.containsIgnoreCaseMultipleWords(i.getTitle() + " " + i.getId(), queries))
```

Comment:
Maybe we could replace this long function name with an overload instead? Kind of like extending the `contains` operation to lists.